Streetcode/Streetcode.DAL/Repositories/Interfaces/Media/IAudioRepository.cs

```diff
@@ -0,0 +1,10 @@
+﻿
+namespace Repositories.Interfaces;
+
+public interface IAudioRepository
+{
+    public void GetAudioAsync();
+    public void UploadAudioAsync();
+    public void DeleteAudioAsync();
```

Comment:
I think we can remove these duplicated methods from all repository's interfaces and just inherit them from IRepositoryBase<T>